cirq-core/cirq/experiments/xeb_fitting.py

```diff
@@ -146,6 +146,47 @@ def get_initial_simplex_and_names(
         """Return an initial Nelder-Mead simplex and the names for each parameter."""
 
 
+def _try_defaults_from_unitary(gate: 'cirq.Gate') -> Optional[Dict[str, 'cirq.TParamVal']]:
+    u = protocols.unitary(gate, default=None)
+    if u is None:
+        return None
+
+    gamma = np.angle(u[1, 1] * u[2, 2] - u[1, 2] * u[2, 1]) / -2
+    phi = -np.angle(u[3, 3]) - 2 * gamma
+    phased_c_theta_2 = u[1, 1] * u[2, 2]
+    if phased_c_theta_2 == 0:
```

Comment:
I guess `c` here is for `cos` but it is no clear from name. Also, why not just `if u[1, 1] * u[2, 2] == 0`?

Reply:
> I guess `c` here is for `cos` but it is no clear from name. Also, why not just `if u[1, 1] * u[2, 2] == 0`?

to signal what the quantity is .... changed to `phased_cos_theta_2`